crypto/rand/rand_lcl.h

```diff
@@ -203,7 +207,16 @@ struct rand_drbg_st {
 /* The global RAND method, and the global buffer and DRBG instance. */
 extern RAND_METHOD rand_meth;
 
-/* How often we've forked (only incremented in child). */
+/*
+ * A "generation count" of forks.  Increment-only, and only ever written to
```

Comment:
Your focus is so much on the fork safety that you 'forget' to state the essential:

```
* A "generation count" of forks. Incremented in the child process after a fork.
*
* Since rand_fork_ount is increment-only, and only ever written to ...
```

Reply:
I would like to see that enhancement too.